### PR TITLE
Add AddJob command to the User Guide

### PR DESCRIPTION
**Pull request by seanleong339**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -196,44 +196,34 @@ Example:
...
 
-`l/[LOCATION]`: Location of the job
+`a/[ADDRESS]`: Location of the job 
 
-`s/[SALARY]`:  Salary of job, based on how much the job pays a month. The SALARY must be a positive integer
-1000,2000,3000,...
+`q/[QUALIFICATION]`: Education qualification required for the job
 
-`sp/[SPECIALIZATION]`: Field of Specialisation required for the job
+`pos/[POSITION]`: Type of job, whether part-time or full time.
```
Comment (Justinnje) [inline docs/UserGuide.md:214]: I think we cna specify here that there are only 2 valid inputs? Maybe similar to my PR?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -196,44 +196,34 @@ Example:
...
 
-`d/[DURATION]`: Duration of job in months. ie d/8 represents 8 months
+`s/[SALARY]`:  Salary of job, based on how much the job pays a month. The SALARY must be a positive integer
```
Comment (Justinnje) [inline docs/UserGuide.md:216]: Should state that this is a range. Something similar to my PR?

**Review by Justinnje — COMMENTED** [review]
LGTM just some changes

File: docs/UserGuide.md (hunk 1 above)
Comment (seanleong339) [reply docs/UserGuide.md:214]: Ok added it as a note

File: docs/UserGuide.md (hunk 2 above)
Comment (seanleong339) [reply docs/UserGuide.md]: Ok added it as a note

**Review by Justinnje — APPROVED** [review]
LGTM

**Pull request merged**